driver-sync/src/test/functional/com/mongodb/client/CrudTest.java

```diff
@@ -80,11 +80,6 @@ public void shouldPassAllOutcomes() {
             expectedResult.asDocument().append("upsertedCount", actualResult.asDocument().get("upsertedCount"));
         }
 
-        // Remove insertCount
```

Comment:
Now properly handling this in JsonPoweredCrudTestHelper, so this isn't needed anymore.